src/components/Greeting.jsx

```diff
@@ -0,0 +1,18 @@
+const DefaultGreeting = () => {
+  var myDate = new Date();
+  var hours = myDate.getHours();
+  var greet;
+
+  if (hours < 12) greet = 'morning';
+  else if (hours >= 12 && hours <= 17) greet = 'afternoon';
+  else if (hours >= 17 && hours <= 24) greet = 'evening';
```

Comment:
Do you want to try to put this in a helper/utility function? 

Something like
```
export const getDescriptiveTimeOfDay = () => {
// code above goes here
return greet;
}
```

Then you can choose to call the function inline or assign it to a variable and call that. 

I also highly suggest that you use `let` instead of `var` to avoid scoping issues and write it with the curly brackets for better readability.

Reply:
Initially, that's how I had it set up, in its own function but wasn't sure how to pass the greeting time into the default function. I will mess around with this and see if I can figure it out.